pkg/interfaces/contracts/vault/IVaultAdmin.sol

```diff
@@ -247,14 +248,12 @@ interface IVaultAdmin {
      * @param wrappedToken Address of the wrapped token that implements IERC4626
      * @param sharesToRemove Amount of shares to remove from the buffer. Cannot be greater than sharesOwner's
      * total shares
-     * @param sharesOwner Address that owns the deposited liquidity.
      * @return removedUnderlyingBalanceRaw Amount of underlying tokens returned to the user
      * @return removedWrappedBalanceRaw Amount of wrapped tokens returned to the user
      */
```

Comment:
There are some stale comments in IVaultAdmin, such as initializeBuffer and addLiquidityToBuffer, where it says "...an yield-bearing buffer (one of the Vault's internal ERC4626 buffers)"

Should be:
"an internal ERC4626 buffer in the Vault"